client/src/components/questions/Answer.jsx

```diff
@@ -74,11 +84,18 @@ const Answer = ({ answerData }) => {
     }
   }
 
+  var imageClick = (e) => {
+    setModalURL(e.target.src);
+    setImageModal(true);
+  }
+
```

Comment:
Every time I read your code I learn something about hooks I didn't know. Like you can setImageModal(true) rather than setImageModal(imageModal = true). Good to know.